### ISSUE TITLE
MusicXML converter

### ISSUE THREAD

**Issue by tonyfung99** [jianpu/musje-123-notation#2]
Hey, 

Would u consider to do a MusicXML parser so that can convert between normal music notation to 簡譜?

**Comment by malcomwu** [comment]
Yes, I did think about the MusicXML converter (import/export). We don't need to make a new parser because XML parser/serializer and XML DOM exist for all major browsers. The converter should convert the musje model and the XML DOM into one another.
